test/parallel/test-fs-promises-file-handle-read.js

```diff
@@ -68,6 +68,27 @@ async function validateReadNoParams() {
   await fileHandle.read();
 }
 
+/**
+ * Validates that the zero position is respected after the position has been
+ * moved. The test iterates over the xyz chars twice making sure that the values
+ * are read from the correct position.
+ */
```

Comment:
Just a nit for consistency

```suggestion
// Validates that the zero position is respected after the position has been
// moved. The test iterates over the xyz chars twice making sure that the values
// are read from the correct position.
```